src/Persistence/LocalStorage.ts

```diff
@@ -0,0 +1,32 @@
+import {FallbackStorage} from '@conclurer/local-storage-fallback';
+
+export module LocalStorage {
+    // Auto detect supported storage adapter (default behavior)
+    let store = new FallbackStorage();
+
+    /**
+     * @param {String} key - a key for a stored object
+     * @return {Object} a promise for the object at the given key
+     */
+    export const getItem = (key: string): Promise<Object> => {
+        return new Promise((resolve, reject) => {
+            let itemAsString = store.getItem(key)
+            try {
+                resolve(JSON.parse(itemAsString))
+            } catch (e) {
+                resolve(itemAsString)
```

Comment:
What would trigger JSON.parse to fail if we're stringifying everything that we store?